### DISCUSSION TITLE
Unable to verify student developer pack

### DISCUSSION THREAD

**Discussion by Mukulkumar057** [community/community#50104] [GitHub Education]
I have tried 3 times with my college ID card and the valid date is also mentioned in the ID card still my application got rejected 3 times. What should I do now?
![Screenshot_2023-03-15-15-56-49-59_e307a3f9df9f380ebaf106e1dc980bb6](https://user-images.githubusercontent.com/59232776/225281674-901c9950-67d9-4ec0-92fc-2f0f59c2449c.jpg)

**Comment by Ntshangase** [comment]
Maybe you missed something during the application, I would suggest reading the documentation provided and try to resubmit your application.

You can also check out: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student

**Reply by Ntshangase** [reply]
Hey @Mukulkumar057, Just make sure that on the document you are submitting it doesn't say " 2022 -2023 " , I think you need something which says "2023 -2024" for example.

**Reply by Mukulkumar057** [reply]
I was trying to submit my ID card of my University and on that card it is mentioned "Valid till : 2023".

**Reply by Ntshangase** [reply]
Try checking your student email.

**Reply by Mukulkumar057** [reply]
I'm using my official University ID with edu.in domain. My University ID is also verified.

**Comment by samus-aran** [comment]
Hi @Mukulkumar057 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by Mukulkumar057** [reply]
![Screenshot_2023-03-17-14-21-59-17_e307a3f9df9f380ebaf106e1dc980bb6](https://user-images.githubusercontent.com/59232776/225857469-4bf87853-331a-470d-b06d-945b4df5f5a3.jpg)
As I mentioned I tried 3 times every time I got this mail as response. And I'm also using my official University ID with edu.in domain.

**Comment by MidKnightXI** [comment]
I have the same problem, tried at least six times with 2 times the same document nothing looks like it's working.
I never had any problem until this year to re-apply to github edu but this thing is so annoying
